src/main/java/com/ironz/unsafe/UnsafeAndroid.java

```diff
@@ -356,6 +403,14 @@ public long allocateMemory(long bytes) {
      * @since 1.8
      */
     @TargetApi(24)
+    public long reallocateMemory(long address, long bytes) {
```

Comment:
Can't find this method in last sources https://android.googlesource.com/platform/libcore/+/android-o-preview-1/ojluni/src/main/java/sun/misc/Unsafe.java